### ISSUE TITLE
Running tkey-verification against an incorrect device gives a 404-error

### ISSUE THREAD

**Issue by cibomahto** [tillitis/tkey-verification#24]
What happened: 

I was trying to verify an old prototype device, with an incorrect USB VID/PID. I used the --port option to force communication to the device.

```
tkey-verification --port /dev/ttyACM0 verify
TKey UDI: 0x0001020304050607(BE) VendorID: 0x0010 ProductID: 8 ProductRev: 3
HTTP GET status code: 404 (404 Not Found)
I/O FAILED: I/O error
```

Expected result:

I expected a message saying that this wasn't a genuine TKey

**Comment by dehanj** [comment]
I agree, it would be better to filter out the 404 Not Found error and treat it in a better way.

Maybe it should be even more verbose, "Not genuine. You are trying to verify a TKey with an UDI not provisioned by Tillitis".
